fusion-plugin-react-router/README.md

```diff
@@ -174,6 +174,23 @@ app.register(GetStaticContextToken, (ctx: Context) => {
 });
 ```
 
+##### `BrowserHistoryToken`
```

Comment:
@ganemone – I made my best efforts here in terms of the docs. Let me know if you have feedback.